paper/paper.md

```diff
@@ -57,7 +49,7 @@ Microsoft Azure, which is the focus of this work, offers tools for managing cost
 These tools are designed for individual subscriptions, and they do not scale effectively for organisations with many subscriptions.
 Moreover, they do not offer a mechanism to impose strict limits on spending or specify the duration for which a budget is valid.
```

Comment:
Maybe could make it really clear that an "Azure Budget" is "soft" - you can overspend, and this may mean a user is consuming more budget than they have.